### DISCUSSION TITLE
[STM32H750] Install MPy: Configuration and Mboot Modify problem.

### DISCUSSION THREAD

**Discussion by freude916** [micropython/micropython#10891] [STM32 / Pyboard]
I want to build micropython on my board.

Unluckily, my board is a STM32H750XBHX (produced by '*Embedfire*'). Yes, it just have 128k internal ROM, and without mpy support.

I' ve saw  #7127 and #10290. The two says "**it's a matter of building a configuration for a H750**" and "**modify mboot so that it memory maps the XIP before jumping to the application**".

As for configuration, I select the project [H7XX Defs](/mcauser/MCUDEV_DEVEBOX_H7XX_M), does it really works?

but boot, how to modify mboot for SPIFlash? I have no idea about it.

I don't have experience about developing in MCUs, so I can't really understand what do these mean. Could anyone please help me with some **detailed** answers?

***(sorry for poor English.)***

**Comment by jahr** [comment]
Hello, 

> As for configuration, I select the project [H7XX Defs](/mcauser/MCUDEV_DEVEBOX_H7XX_M), does it really works?

It seems to me that the mentioned configuration is rather out-of-date and it worked only for H743.

> I' ve saw #7127 and #10290.
> 
> but boot, how to modify mboot for SPIFlash? I have no idea about it.

In #10290 I provided two working configurations for H750 with mboot, I've just tested building and flashing FK750M3 and it still works ;-)

However, if you are using [this](https://doc.embedfire.com/products/link/zh/latest/mcu/stm32/ebf_stm32h750_pro/download/stm32h750_pro.html) board, I see a possible problem. It seems that the board is using dualbank QSPI configuration (similar to STM32H750B_DK) and I couldn't get it to work :-(
If it is not the case you should be able to make it work ;-)

**Reply by freude916** [reply]
Oh, unluckily, I just exactly buy the _Embedfire_ board. :-(

By the way, It seems like you are Chinese too? If so, 老乡，用母语说话。
